R/tibble.R

```diff
@@ -31,9 +31,10 @@ use_tibble <- function() {
   check_is_package("use_tibble()")
   check_uses_roxygen("use_tibble()")
 
-  use_dependency("tibble", "Imports")
-  roxygen_ns_append("@importFrom tibble tibble") && roxygen_remind()
+  created <- use_import_from("tibble", "tibble")
```

Comment:
Should we rebuild the NAMESPACE?

Reply:
I think `use_import_from()` does what you're thinking already

Reply:
Ooooh yeah. I was just confused by the `created`